foreman.te

```diff
@@ -203,6 +192,7 @@ corecmd_exec_ls(foreman_rails_t)
 # The following is needed for "scl enable" which creates temporary
 # shell script /var/tmp/xxxxxxx and a subprocess.
 files_manage_generic_tmp_files(foreman_rails_t)
+files_manage_generic_tmp_dirs(foreman_rails_t)
```

Comment:
I was seeing this denial, I haven't investigated much it looks like SCL is the culprit.